src/Cli/dotnet/commands/dotnet-sln/LocalizableStrings.resx

```diff
@@ -184,9 +184,9 @@
     <value>.slnx file {0} generated.</value>
   </data>
   <data name="CannotMigrateSlnx" xml:space="preserve">
-    <value>Only .sln files can be migrated to .slnx format.</value>
+    <value>Cannot migrate .slnx file.</value>
   </data>
   <data name="SerializerNotFound" xml:space="preserve">
-    <value>Could not read solution file {0}. Supported files are .sln and .slnx valid solutions.</value>
+    <value>Could not find serializer for file {0}.</value>
```

Comment:
The new strings are valid too, but probably the original strings are clearer, so they are preferred. If you disagree, let me know :) 

You don't need to fix the history, you could just change this in a new commit to what it previously was so that it doesn't get changed.

Reply:
Missed this comment, I will include the updated translations in my next commit. Thanks both! :D